data/download_reference_genome.sh

```diff
@@ -3,6 +3,7 @@
 # to download reference genome
 # ./download_reference_genome.sh
 
+cd data
```

Comment:
bu script ile hata alırız. readme dosyasında 84 - 87 satırlar arasına bakarsan, zaten referans genomu indirmek için `cd data` yazıyoruz. `download_reference_genome.sh` dosyasında da `cd data` var. 

yani önce data klasörüne girip, bu betiği çalıştırırsan, betik önce `data` klasörü içerisinde girmeye çalışacak. 

o yüzden buradaki `cd data` komutunu sil bence.

Reply:
cd data silindi